### PR TITLE
Add external identifiers to contact - ISO 19115 2018 AMD

### PR DESCRIPTION
**Pull request by timothypage**

### CODE REVIEW

File: schema/contact.json [hunk 1]
```diff
@@ -32,6 +32,16 @@
         ]
       }
     },
+   "externalIdentifiers": {
```
Comment (hmaier-fws) [inline schema/contact.json:35]: @dwalt I think we should use "externalIdentifier" (singular) to remain consistent with the existing definitions.

File: schema/contact.json [hunk 2]
```diff
@@ -32,6 +32,16 @@
...
+      "type": "array",
+      "description": "Identifier for a contact, such as ORCid, RoR",
+      "translation": {
+        "ISO 19115-1": ["CI_ResponsibleParty > partyIdentifier"]
```
Comment (hmaier-fws) [inline schema/contact.json:39]:
```suggestion
        "ISO 19115-1": ["CI_ResponsibleParty > partyIdentifier"],
        "FGDC CSDGM": ["no translation"]
```

File: schema/contact.json [hunk 3]
```diff
@@ -32,6 +32,16 @@
         ]
       }
     },
+   "externalIdentifiers": {
+      "type": "array",
+      "description": "Identifier for a contact, such as ORCid, RoR",
```
Comment (hmaier-fws) [inline schema/contact.json:37]: @dwalt  Recommend using ORCID iD and ROR as per recommendations from maintaining organizations.
  - https://info.orcid.org/brand-guidelines/#the-orcid-name
  - https://ror.org/display-guidelines/#ror-glossary

File: schema/contact.json (hunk 1 above)
Comment (dwalt) [reply schema/contact.json:35]: I agree with the change.

File: schema/contact.json (hunk 2 above)
Comment (dwalt) [reply schema/contact.json:39]: Agreed

**Review by dwalt — COMMENTED** [review]
Users would like namespace to be optional. To clarify, will this validation still return a warning message, rather than an invalid status?

File: schema/contact.json (hunk 3 above)
Comment (dwalt) [reply schema/contact.json:37]: @hmaier-fws To clarify, the description would look like this: "Identifier for a contact, such as ORCID ID or ROR"?

Comment (hmaier-fws) [reply schema/contact.json:37]: Yes, except "ORCID iD" (lowercase "i")

File: schema/identifier.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
       "ISO 19115-2": ["MD_Identifier"]
     },
     "type": "object",
-    "required": ["identifier"],
+    "required": ["identifier", "namespace"],
```
Comment (hmaier-fws) [inline schema/identifier.json:11]: @dwalt was "namespace" supposed to be optional?

**Comment by dwalt** [comment]
Yes, per user request. A warning status would not require entry. Are you wanting to leave it completely optional, without a warning?
________________________________
From: Hilmar Maier ***@***.***>
Sent: Tuesday, August 2, 2022 5:06 PM
To: adiwg/mdJson-schemas ***@***.***>
Cc: Walworth, Dennis H ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [adiwg/mdJson-schemas] Add external identifiers to contact - ISO 19115 2018 AMD (PR #257)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@hmaier-fws commented on this pull request.

________________________________

In schema/identifier.json<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fadiwg%2FmdJson-schemas%2Fpull%2F257%23discussion_r936149217&data=05%7C01%7Cdwalworth%40usgs.gov%7Cbbd86c0b817d41f6486b08da74ec80d3%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637950856372263906%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=f%2BMkLVJSg%2FQIE2e7ziLOd16L5jcW49m61oZetRSHGoE%3D&reserved=0>:

> @@ -8,7 +8,7 @@
       "ISO 19115-2": ["MD_Identifier"]
     },
     "type": "object",
-    "required": ["identifier"],
+    "required": ["identifier", "namespace"],


@dwalt<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdwalt&data=05%7C01%7Cdwalworth%40usgs.gov%7Cbbd86c0b817d41f6486b08da74ec80d3%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637950856372263906%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=A6cwdhWIqj4m3efBC6cFt0GAVUEnNE3MdrcdcD07s%2Bc%3D&reserved=0> was "namespace" supposed to be optional?

**Review by hmaier-fws — CHANGES_REQUESTED** [review]
We should use singular names to remain consistent with what the project has been using ("externalIdentifier" instead of "externalIdentifiers").

We also recently decided to include a FGDC example wherever an ISO example is provide to (or indicate "no translation") so that it is clear to users that we didn't forget something. I have added the FGDC example to the code. 

I thought that the identifier namespace was supposed to be optional, I'll let @dwalt chime in on that.

File: schema/identifier.json (hunk 4 above)
Comment (dwalt) [reply schema/identifier.json:11]: @hmaier-fws Yes, per user request. A warning status would not require entry. Are you wanting to leave it completely optional, without a warning?

**Review by dwalt — CHANGES_REQUESTED** [review]
Sorry to nitpick, in the description: ORCiD rather than ORCid and ROR rather than RoR, per Hilmar's references.

**Comment by timothypage** [comment]
> Sorry to nitpick, in the description: ORCiD rather than ORCid and ROR rather than RoR, per Hilmar's references.

No worries. I thought I had made that change but I guess not!  thanks for catching.

**Pull request merged**